keadm/cmd/keadm/app/cmd/edge/reset_others.go

```diff
@@ -83,56 +76,45 @@ func NewOtherEdgeReset() *cobra.Command {
 				}
 			}
 
-			staticPodPath := ""
+			step.Printf("clean up static pods")
 			config, err := util.ParseEdgecoreConfig(common.EdgecoreConfigPath)
 			if err != nil {
-				fmt.Printf("failed to get edgecore's config with err:%v\n", err)
+				klog.Warningf("failed to parse edgecore configuration, skip cleaning up static pods, err: %v", err)
 			} else {
 				if reset.Endpoint == "" {
 					reset.Endpoint = config.Modules.Edged.TailoredKubeletConfig.ContainerRuntimeEndpoint
 				}
-				staticPodPath = config.Modules.Edged.TailoredKubeletConfig.StaticPodPath
-			}
-			// first cleanup edge node static pod directory to stop static and mirror pod
-			if staticPodPath != "" {
-				if err := phases.CleanDir(staticPodPath); err != nil {
-					fmt.Printf("Failed to delete static pod directory %s: %v\n", staticPodPath, err)
-				} else {
-					time.Sleep(1 * time.Second)
-					fmt.Printf("Static pod directory has been removed!\n")
+				staticPodPath := config.Modules.Edged.TailoredKubeletConfig.StaticPodPath
+				if staticPodPath != "" {
+					if err := phases.CleanDir(staticPodPath); err != nil {
+						klog.Warningf("failed to delete static pod directory %s: %v", staticPodPath, err)
+					}
 				}
 			}
-
-			// 1. kill edgecore process.
-			// For edgecore, don't delete node from K8S
+			step.Printf("kill edgecore and remove edgecore service")
 			if err := TearDownEdgeCore(); err != nil {
 				return err
 			}
-
-			// 2. Remove containers managed by KubeEdge. Only for edge node.
+			step.Printf("clean up containers managed by KubeEdge")
 			if err := util.RemoveContainers(reset.Endpoint, utilsexec.New()); err != nil {
-				fmt.Printf("Failed to remove containers: %v\n", err)
+				klog.Warningf("failed to clean up containers, err: %v", err)
 			}
-
-			// 3. Clean stateful directories
-			if err := util.CleanDirectories(isEdgeNode); err != nil {
-				return err
+			step.Printf("clean up dirs created by KubeEdge")
+			if err := util.CleanDirectories(true); err != nil {
```

Comment:
```suggestion
			if err = util.CleanDirectories(true); err != nil {
```
It seems that the variable `err` is already defined at line 80

Reply:
This keeps err scoped to the If block. :-)